repos/spack_repo/builtin/packages/paraview/package.py

```diff
@@ -308,13 +307,25 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("xz")
     depends_on("zlib-api")
     depends_on("libcatalyst@2:", when="+libcatalyst")
-    depends_on("hip@5.2:", when="+rocm")
+
+    # For @6.0:, depend on external Viskores module when needed.
+    depends_on("viskores", when="@6: +fides")
+    depends_on("viskores +cuda", when="@6: +cuda")
+    for arch in CudaPackage.cuda_arch_values:
+        depends_on(f"viskores +cuda cuda_arch={arch}", when=f"@6: +cuda cuda_arch={arch}")
+    depends_on("viskores +rocm", when="@6: +rocm")
+    for target in ROCmPackage.amdgpu_targets:
+        depends_on(
+            f"viskores +rocm amdgpu_target={target}", when=f"@6: +rocm amdgpu_target={target}"
+        )
+
+    depends_on("hip@5.2:", when="@:5.13 +rocm")
     # CUDA thrust is already include in the CUDA pkg
-    depends_on("rocthrust", when="@5.13: +rocm ^cmake@3.24:")
+    depends_on("rocthrust", when="@5.13 +rocm ^cmake@3.24:")
     for target in ROCmPackage.amdgpu_targets:
         depends_on(
             "kokkos@:3.7 +rocm amdgpu_target={0}".format(target),
-            when="+rocm amdgpu_target={0}".format(target),
+            when="@:5.13 +rocm amdgpu_target={0}".format(target),
```

Comment:
You should be able to just use `@:5` here. I think that is more clear to read "All versions of 5 and older`

```suggestion
            when="@:5 +rocm amdgpu_target={0}".format(target),
```